src/systems/filecoin_mining/storage_mining/storage_mining_subsystem.go

```diff
@@ -206,13 +207,21 @@ func (sms *StorageMiningSubsystem_I) VerifyElectionPoSt(header block.BlockHeader
 		return false
 	}
 
-	pow, err := sma._getActivePower()
-	if err != nil {
+	activeSectorWeight, inactiveSectorWeight, ok := spa.GetSectorWeightForMiner(header.Miner())
+	if !ok {
+		// TODO: better error handling
+		return false
+	}
+
+	currPledge, ok := spa.GetCurrPledgeForMiner(header.Miner())
+	if !ok {
 		// TODO: better error handling
 		return false
 	}
 
-	if !spa.ActivePowerMeetsConsensusMinimum(pow) {
+	pow := inds.BlockReward_StoragePower(activeSectorWeight, inactiveSectorWeight, currPledge)
```

Comment:
minor: I might rename this (mapping sector params to power doesn't necessary have to do with block reward)

Reply:
done, calling it inds.StoragePower for now